share/spice/uv/uv.js

```diff
@@ -0,0 +1,50 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_uv = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || !api_result.length || api_result.length !== 1) {
+            return Spice.failed('uv');
+        }
+
+        //extract city and date from first result
+        var result = api_result[0];
```

Comment:
I wanted to shield against 0 results being returned. Also in the next version there might be a list of hourly uv index values. Not sure what you mean by relevancy check for more than 1 result?